single_kernel_mongo/managers/mongo.py

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/env python3
```

Comment:
`mongod_ready` is a bit more generic.
`initialise_operator_user` might need to be changed back to former behavior because of kubernetes.

Big changes of logic for managing users and db : get rid of `oversee_users` to go to something more relation-oriented (handle one relation at a time).
Drop some useless and redundant methods.
Cleaner and faster code.